object_database/web/cells/non_display/key_action.py

```diff
@@ -0,0 +1,41 @@
+"""KeyAction Cell
+
+Listens key-combo press events from the UI"""
+from ..cells import Cell
+
+
+class KeyAction(Cell):
+    """
+    Parameters
+    __________
+    keyCmd str: A string of a key combo like `Alt+i`, `Meta+D`
+                or event single keys like `X`.
```

Comment:
Is the format of strings that are used to describe key combinations standardized (i mean is it a standard web thing)? If so, can we include a link. If not can we describe in detail what will work? Also, how does this work mac vs pc? Also, how to encode arrow keys?

Reply:
The `<metakey>+<alphanum>` structure is non standard, but the practice seems to be common enough. If you would like I can add this to the comments somewhere in the Cell module.
  
The possible key values -- ie the parts after a `+` or just represented on their own, can be found on MDN here:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key
  
For arrows the names are `ArrowRight` `ArrowLeft` `ArrowUp` `ArrowDown`
  
Mac vs PC is tough for me to test out -- I've never been able to get this repository to build and install on Mac. Daniel and I tried early on but we were spinning our wheels so left that effort behind